spec/models/note_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe Note, type: :model do
+  include ActiveSupport::Testing::TimeHelpers
+  let(:author){Fabricate(:user, email: "admin@art.sy")}
+  let(:submission){Fabricate(:submission)}
+  let(:note){ Note.create(submission: submission, created_by: author.id, body: "Im a note") }
+  
+
+  it 'belongs to an author' do
+    note = Note.new(submission: submission, created_by: author.id, body: "Im a note")
+
+    expect(note).to be_valid
+    expect(note.author.email).to eq "admin@art.sy"
+  end
+
+  it 'requires a body' do
+    note = Note.new
+    expect(note).not_to be_valid
+    expect(note.errors.messages[:body]).to eq ["can't be blank"]
+    
+    note.body = ''
+    expect(note).not_to be_valid
+    expect(note.errors.messages[:body]).to eq ["can't be blank"]
+
+    note.body = "f"
+    expect(note).to be_valid
```

Comment:
IDK how I feel about this pile of assertions. Validating presence is pretty straightforward and so this was mostly me just confirming my understanding of it.

Reply:
I wouldn't test Rails validations that do not involve any conditionals.